packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -938,7 +938,44 @@ export const components = ({
             margin: 0,
             marginInlineEnd: odysseyTokens.Spacing1,
           },
+          ...(ownerState.variant === "lamp" && {
+            paddingBlock: 0,
+            paddingInline: 0,
+            borderRadius: 0,
+            border: 0,
+            backgroundColor: "transparent",
+            color: odysseyTokens.TypographyColorBody,
 
+            "&::before": {
+              content: "''",
```

Comment:
This is weird. Why are there quotes in `content`?

Reply:
(disclosure: this was copied from what we had implemented a few months ago)

This method is new to me too, I normally would add a space but the rationale I found was that the single quotes in `content: "''";` is interpreted as empty whereas using a space adds one character of whitespace.